### ISSUE TITLE
improper path to item if nested...

### ISSUE THREAD

**Issue by bdillahu** [Sommerregen/grav-plugin-archive-plus#2]
I have a collection of items in a /blog subfolder:

---
01.stuff
02.blog
   item 1
   item 2
   item 3
---

when the archive is dealing with the blog stuff, the links for the item itself looks like

http://www.site.com/blog/blog/item 1

Note the duplicated "blog"

The links for the 2015 (year) and January (month) or whatever are correct, with only one "blog"

I probably have something loused up... help?

Thanks

**Comment by Sommerregen** [comment]
Hard to tell... I have a similiar structure like yours and this doesn't show up. Maybe your home alias is different than "/blog"? However you may copy `user/plugins/archive_plus/templates/partials/archive_plus.html.twig` into your custom theme (into `templates/partials/archive_plus.html.twig`) and change [line 81](https://github.com/Sommerregen/grav-plugin-archive-plus/blob/master/templates/partials/archive_plus.html.twig#L81) to

```twig
<a href="{{ item.url }}"{{ active }}>{{ item.title }}</a>
```

Let me know if that works.

**Comment by bdillahu** [comment]
Ah, thanks! Found the line but wasn't sure what to use... that works for me.

My home alias is different (the blog section isn't the "home page" of the site).

Another related question. If I have "month" enabled, clicking on the "month" doesn't really do anything (it doesn't display only items in that month or whatever. Is it supposed to?

Thanks,
Bruce

**Comment by Sommerregen** [comment]
Well, it depends, whether you have set `show.items` to "true" or not. I think you want something like this ![example](https://github.com/Sommerregen/grav-plugin-archive-plus/blob/master/assets/screenshot_3.png) or [this](https://github.com/Sommerregen/grav-plugin-archive-plus/blob/master/assets/screenshot_4.png). Then, the configuration shall looks like

```yaml
show:
    counter: true    # either true or false for you
    year: false      # no year
    month: true      # show month
    items: true      # show items
```

When `show.items : false` no items are visible and clicking on the month directs you to the "archive month page". I will test it as soon as I have a machine running. :-)

**Comment by bdillahu** [comment]
Changing show.items doesn't seem to make the "month page" view work for me...

**Comment by Sommerregen** [comment]
No, this shouldn't be the problem. Your problem seems to be more complicated I guess. But I don't know what is going wrong. Internally this plugin uses taxonomys to display a collection of pages (for a month or for a year). When there is no difference, it either means there are no other items or the tag isn't set correctly to the page.

Are you using the latest Grav release **v0.9.29**? Did a cache clear help? If not can you send me your site either via mail (see the sourcecode for my mail address) or upload it somewhere and post the link here?

**Comment by bdillahu** [comment]
OK, after digging through the code, I found it, I believe.

I hadn't just added a "category" for archive items, but added a separate taxonomy for items that would appear.

I had managed to pick the (obvious :-)) name of 'archive' - which is also used in your code :-)

If I change my taxonomy name to "archive_this" it all starts working as nearly as I can tell.

Thanks for all your help!

**Comment by Sommerregen** [comment]
Ok. I hope that I'm not misunderstanding. But usually there is no need to add a taxonomy (here: `archive`) to any page. This plugin will do it automatically. You can convince yourself be deleting such taxonomies. However by default **Archive Plus** is looking for pages which are associated to a category (see [here](https://github.com/Sommerregen/grav-plugin-archive-plus/blob/master/archive_plus.yaml#L23)). In other words in each page header you have to set at least

```yaml
taxonomy:
  category: blog
```

**Comment by bdillahu** [comment]
Yes, for other reasons I didn't want to change or add to my category taxonomy for this data, so I had done:

taxonomy:
  archive: true
  category: XXXX

and then in the archive_plus.yaml had:

filter:
  archive: true

----

That doesn't work so well :-)

Changing the name to archive_this it works.

**Comment by Sommerregen** [comment]
Fixed in version **Archive Plus v1.2.0**.
